src/Cli/TerminusCommands.php

```diff
@@ -0,0 +1,158 @@
+            throw new \Exception("Either commands or releases must be updated.");
+        }
+
+        $terminusRepo = $options['terminus-repo'];
+        $terminusRelease = $options['terminus-release'];
+
+        $terminusDir = sys_get_temp_dir() . '/update-tool/terminus/';
+        if (!is_dir($terminusDir)) {
+            mkdir($terminusDir, 0755, true);
+        }
+
+        if ($terminusRelease === 'latest') {
+            $terminusRelease = $this->getLatestRelease($api, $terminusRepo);
+        }
+        $downloadUrl = 'https://github.com/pantheon-systems/terminus/releases/download/' . $terminusRelease . '/terminus.phar';
+
+        file_put_contents($terminusDir . 'terminus.phar', file_get_contents($downloadUrl));
+        if (!file_exists($terminusDir . 'terminus.phar')) {
+            throw new \Exception(sprintf('Failed to download terminus.phar from release %s.', $terminusRelease));
+        }
+        chmod($terminusDir . 'terminus.phar', 0755);
+
+        $githubRepo = $options['github-repo'];
+        $baseBranch = $options['base-branch'];
+
+        $branchNamePrefix = $options['branch-name-prefix'];
+        $branchName = $branchNamePrefix . date('Ymd');
+
+        $url = 'git@github.com:' . $githubRepo . '.git';
+        $remote = new Remote($url);
+        $dir = sys_get_temp_dir() . '/update-tool/' . $remote->project();
+        $workingCopy = WorkingCopy::cloneBranch($url, $dir, $baseBranch, $api);
+        $workingCopy->setLogger($this->logger);
+        $workingCopy->createBranch($branchName, $baseBranch, true);
+
+        if ($updateCommands) {
+            $this->logger->info('Updating Terminus commands...');
+            exec(sprintf("cd %s && %s/terminus.phar list --format=json > source/data/commands.json", $dir, $terminusDir), $output, $return);
+            if ($return != 0) {
+                throw new \Exception(sprintf('Failed to list terminus commands. Output: %s', implode("\n", $output)));
+            }
+
+            $commands = json_decode(file_get_contents($dir . '/source/data/commands.json'), true);
+            $commandsJson = json_encode($commands, JSON_PRETTY_PRINT);
+
+            // Adjust output.
+            $commandsJson = str_replace(
+                [
+                'site_env',
+                'drush_command',
+                'wp_command',
+                ],
+                [
+                    'site.env',
+                    'command',
+                    'command',
+                ],
+                $commandsJson
+            );
+
+            file_put_contents($dir . '/source/data/commands.json', $commandsJson);
+        }
+        if ($updateReleases) {
+            $this->logger->info('Updating Terminus releases...');
+            $releases = $this->getAllReleases($api, $terminusRepo);
+            $releasesJson = json_encode($releases, JSON_PRETTY_PRINT);
+            file_put_contents($dir . '/source/data/terminusReleases.json', $releasesJson);
+        }
+
```

Comment:
Check to see if the lines above modified any files; if not, log a "nothing to do" message and exit without creating a PR.

Reply:
I did this; however, terminusReleases contains downloadCount info so unless --no-update-releases specified it will always change

Reply:
We should strip the download count then.

Reply:
Or, safer, set the download count to `0` in each entry.

Reply:
download count set to 0.